### DISCUSSION TITLE
Kimai 2.20.1 Installation Inquiry

### DISCUSSION THREAD

**Discussion by seleondevel** [kimai/kimai#5030] [Plugins]
Hello.

git clone -b 2.20.1 --depth 1 https://github.com/kimai/kimai.git
cd kimai/
computer install --optimize-automatic loader

If you install kimai with the above command, during installation as shown below
A yellow line warning appears.

![스크린샷 2024-08-26 171413](https://github.com/user-attachments/assets/f9350933-472c-4a2f-93de-744f86b06239)

Class API\Authentication\AccessTokenHandlerTest.php located in ./tests/API/Authentication/AccessTokenHandlerTest.php does not comply with the psr-4 autoloading standard (rules: App\Tests\=> ./tests). Skip.
Class API\AuthenticationTest located in ./tests/API/AuthenticationTest.php does not comply with psr-4 auto-loading standard (rules: App\Tests\=> ./tests), skip.

And, if you check the version after installation,

$ php bin/console --version
Kimai 2.20.0 (env: prod, debug: false)

![스크린샷 2024-08-26 171425](https://github.com/user-attachments/assets/a9c6bb2c-8760-45ae-8fa1-37781bce424b)

It pops up to 2.20.0 instead of 2.20.1, is this normal?

What can I do to prevent the warning from opening?

Please check.

**Comment by kevinpapst** [comment]
https://github.com/kimai/kimai/issues/5023
